src/Illuminate/View/ViewServiceProvider.php

```diff
@@ -133,4 +136,16 @@ public function registerBladeEngine($resolver)
             return new CompilerEngine($this->app['blade.compiler']);
         });
     }
+
+    /**
+     * Register the Mix instance.
+     *
+     * @return void
+     */
+    public function registerMix()
+    {
+        $this->app->singleton('mix', function () {
+            return new Mix();
```

Comment:
new classes without parameters are declared without the braces: `return new Mix;`